packages/core/upload/server/services/upload.js

```diff
@@ -235,6 +241,10 @@ module.exports = ({ strapi }) => ({
         }
       }
     }
+
+    // Upload image
+    uploadPromises.push(getService('provider').upload(fileData));
```

Comment:
not sure I understand the reordering 🤔

Reply:
Sanitizing the file name did work for this inmediate issue, but reordering the upload prevented any unhandled errors for some reason, I did not have the time to find which stream was causing the issue. We could take a look together